### PR TITLE
RDCC-953 Nightly Build Fix

### PR DESCRIPTION
**Pull request by ConorH994**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDCC-953

### Change description ###

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/JurisdictionServiceImplTest.java [hunk 1]
```diff
@@ -78,7 +78,10 @@ public void should_create_jurisdictionuserprofilerequest_for_super_user() {
 
         JurisdictionUserCreationRequest request = jurisdictionServiceImpl.createJurisdictionUserProfileRequestForSuperUser(user);
         assertThat(request).isNotNull();
+        assertThat(request.getId()).isNotEmpty();
```
Comment (shreedhar-hmcts) [inline src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/JurisdictionServiceImplTest.java]: is this test case  is required for nightly build to pass?

Comment (ConorH994) [reply src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/JurisdictionServiceImplTest.java]: yeah the mutation score was below threshold

Comment (shreedhar-hmcts) [reply src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/JurisdictionServiceImplTest.java]: ok.

File: config/owasp/suppressions.xml [hunk 2]
```diff
@@ -2,6 +2,42 @@
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
     <suppress>
         <notes><![CDATA[
+   file name: selenium-opera-driver-3.141.59.jar
+   ]]></notes>
+        <sha1>6b11ed01182bfd8bb62ddeae34cf9eadd3cfc7ff</sha1>
+        <cve>CVE-2003-1561</cve>
+        <cve>CVE-2008-1761</cve>
+        <cve>CVE-2008-1764</cve>
```
Comment (abhijitdiwan) [inline config/owasp/suppressions.xml]: can't we use package or regex validation here rather than using too much CVE?

**Comment by shreedhar-hmcts** [comment]
Get second approval by Abhijit or Prashanth

**Pull request merged**